tests/suites/secrets_iaas/vault.sh

```diff
@@ -5,8 +5,7 @@ run_secrets_vault() {
 
 	model_name='model-secrets-vault'
 	juju add-secret-backend myvault vault endpoint="$VAULT_ADDR" token="$VAULT_TOKEN"
-	add_model "$model_name"
-	juju --show-log model-config secret-backend=myvault -m "$model_name"
+	add_model "$model_name" --config secret-backend=myvault
 
```

Comment:
This was a fix I added we should keep.
The `add-model` does not support extra args for model config.
Ah, we actually have to backward port these changes to 3.1.